SU-SWS/stanford_paragraph_types
# HSDO-876 Dynamic Paragraph type

# READY FOR REVIEW

# Summary
- Added dynamic paragraph item that allows for views to be used in nodes.

# Needed By (Date)
- 6/7

# Steps to Test

1. Checkout branch
2. enable stanford_paragraph_types_dynamic
3. use [stanford_story_page](https://github.com/SU-SWS/stanford_story_page) or use a paragraph field on your own entity.
4. choose "Dynamic Content" from the paragraph type options
5. select 1 or more views from the list. 
6. 1 view should fill the area. 2 views should have a span6 each, 3 views have span4 etc. 

# See Also
- [Tests](https://github.com/SU-SWS/linky_clicky/pull/162) - merge clicky branch, then change travis branch here before merging.
